sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -872,12 +875,41 @@ def _analyze_result_callback(
         )
 
     @distributed_trace
-    def begin_analyze_actions(  # type: ignore
+    def begin_analyze_actions(
         self,
-        documents,  # type: Union[List[str], List[TextDocumentInput], List[Dict[str, str]]]
-        actions,  # type: List[Union[RecognizeEntitiesAction, RecognizeLinkedEntitiesAction, RecognizePiiEntitiesAction, ExtractKeyPhrasesAction, AnalyzeSentimentAction, ExtractSummaryAction, RecognizeCustomEntitiesAction, SingleCategoryClassifyAction, MultiCategoryClassifyAction]] # pylint: disable=line-too-long
-        **kwargs  # type: Any
-    ):  # type: (...) -> AnalyzeActionsLROPoller[ItemPaged[List[Union[RecognizeEntitiesResult, RecognizeLinkedEntitiesResult, RecognizePiiEntitiesResult, ExtractKeyPhrasesResult, AnalyzeSentimentResult, ExtractSummaryResult, RecognizeCustomEntitiesResult, SingleCategoryClassifyResult, MultiCategoryClassifyResult, DocumentError]]]]  # pylint: disable=line-too-long
+        documents: Union[List[str], List[TextDocumentInput], List[Dict[str, str]]],
+        actions: List[
+            Union[
+                RecognizeEntitiesAction,
```

Comment:
it might be nice to do a type alias at the top of the page, something like `ActionTypes=Union[...]`, that way it's a little cleaner in the method, and you can share across sync and async

Reply:
Let me give it a try and see if docs/APIview are still happy with that!

Reply:
Seems good. Made the change